.pre-commit-config.yaml

```diff
@@ -25,16 +26,16 @@ repos:
       - id: isort
         name: isort (python)
   - repo: https://github.com/pycqa/bandit
-    rev: 1.7.4
+    rev: 1.7.5
     hooks:
       - id: bandit
-        args: ["-ll", "--skip=B108,B608,B310,B303,B324"]
+        args: ["-ll", "--skip=B108,B608,B310,B303,B324,B113"]
         files: .py$
   - repo: https://github.com/sqlfluff/sqlfluff
-    rev: 1.4.1
+    rev: 2.0.2
     hooks:
       - id: sqlfluff-lint
-        additional_dependencies: ['dbt-bigquery==1.3.0', 'sqlfluff-templater-dbt']
+        additional_dependencies: ['dbt-bigquery', 'sqlfluff-templater-dbt']
```

Comment:
1.3.0 was previously set here because of issues with [breaking changes to exception names beginning in 1.4.0](https://github.com/dbt-labs/dbt-bigquery/pull/460/files) - is that handled?

Reply:
Yes it was fixed in https://github.com/sqlfluff/sqlfluff/pull/4317